README.md

```diff
@@ -100,7 +121,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema provides a model for what information will be included in a new database, and how it will be organized. Within a schema file, you create a table for each entity, define which attributes that entity will have, and describe what data type will be provided for each attribute. Models within the schema can also show associations between tables. For example, each instance in my Apartment table "has_many" tenants, whereas each instance in my Tenant table "belongs_to" an apartment.
```

Comment:
try to avoid the word "model" when talking about a schema. model is specific to ORMs - otherwise great answer!